components/HomePage/index.tsx

```diff
@@ -1,10 +1,197 @@
-import React from "react";
+import React, { useMemo } from "react";
 import { DefaultLayout } from "../../layouts/Default";
+import { KpiOptionIcon, SuccessTokenIcon } from "../Icons";
+import { PrimaryLink, SecondaryLink } from "../Link";
+import SeoHead from "../SeoHead";
+import * as UI from "./components";
+import { hooks, constants } from "../../utils";
+
+const { LINKS, PROJECTS, QUERIES } = constants;
+const { useMediaQuery } = hooks;
 
 export const HomePage: React.FunctionComponent = () => {
   return (
     <DefaultLayout>
-
+      <SeoHead />
+      <UI.Container>
+        <UI.Content>
+          <UI.Title>UMA is an <span>Optimistic Oracle</span> built for web3</UI.Title>
+          <PrimaryLink href={LINKS.docs}>Get started</PrimaryLink>
+        </UI.Content>
+      </UI.Container>
+      <WhatIsOO />
+      <Projects />
+      <WhyUseUMA />
+      <Products />
+      <GetStarted />
     </DefaultLayout>
+  )
+}
+
+const WhatIsOO: React.FunctionComponent = () => {
+  return (
+    <UI.HomeSectionContainer grayBackground>
+      <UI.HomeSectionContent>
+        <UI.HomeSectionHeadline>What is the optimistic oracle?</UI.HomeSectionHeadline>
+        <UI.OODescription>
+          An optimistic oracle services data to smart contracts by assuming it is true unless it is disputed.
+        </UI.OODescription>
+        <UI.OOBoxes>
+          {OOBoxesData.map((data, idx) => (
+            <UI.OOBoxContainer key={idx}>
+              <UI.OOBox>
+                <UI.BoxImgContainer>
+                  <UI.BoxImg src={data.icon} alt="Optimisc Oracle service" />
+                </UI.BoxImgContainer>
+                <UI.OOBoxTextContainer>
+                  <UI.BoxTitle>{data.title}</UI.BoxTitle>
+                  <UI.BoxDescription>{data.description}</UI.BoxDescription>
+                </UI.OOBoxTextContainer>
+              </UI.OOBox>
+            </UI.OOBoxContainer>
+          ))}
+        </UI.OOBoxes>
+        <SecondaryLink href={LINKS.oo}>Learn more</SecondaryLink>
+      </UI.HomeSectionContent>
+    </UI.HomeSectionContainer>
   );
 }
+
+const Projects: React.FunctionComponent = () => {
+  const isTablet = useMediaQuery(QUERIES.laptopAndUp);
```

Comment:
lets use the new dapp development philosophy here too, just to keep things consistent